src/components/Html/Html.jsx

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable react/no-danger */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Html = ({ title, css, scripts, markup, initialState }) => (
+  <html lang="en">
+    <head>
+      <title>{title}</title>
+      <link type="text/css" rel="stylesheet" href={`/${css}`} />
+    </head>
+    <body>
+      <div id="app" dangerouslySetInnerHTML={{ __html: markup }} />
+      <script
+        type="text/javascript"
+        dangerouslySetInnerHTML={{
+          __html: `window.__INITIAL_STATE__ = ${JSON.stringify(initialState)}`,
+        }}
+      />
+      {scripts.map(s => <script key={s} type="text/javascript" src={`/${s}`} />)}
+    </body>
+  </html>
+);
```

Comment:
The HTML is generated with JSX, that allows us to have a better control, and also avoid issues of template literals